resources/application-connector/charts/connector-service/files/certs-restore.sh

```diff
@@ -0,0 +1,23 @@
+set -e
+set -o pipefail
+
+restore() {
+  namespace="$1"
+  secretName="$2"
+  backupSecretName="$secretName-backup"
+
+  echo "Restoring $backupSecretName"
+  secret=$(kubectl -n "$namespace" get secret "$backupSecretName" --ignore-not-found)
+  if [ -n "$secret" ]
+  then
+    echo "Removing $secretName"
+    kubectl delete secret "$secretName" -n="$namespace" --ignore-not-found
```

Comment:
You do not need to delete this secret here. Helm will delete it automatically

Reply:
It it not clear to me how helm should delete it automatically. Could you please elaborate?